src/utils/api.ts

```
@@ -51,6 +53,37 @@ function getCachedStations(): Station[] | null {
 	return cached.data;
 }
 
+// Cache for train data
+const trainCache = new Map<string, { data: Train[]; timestamp: number }>();
+
+// Cleanup function to prevent cache from growing too large
+function cleanupCache(cache: Map<string, any>): void {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)